src/ui/components/SidePanel.tsx

```diff
@@ -100,24 +102,25 @@ export default function SidePanel() {
   const items: any[] = [];
 
   // if (recording?.metadata?.test?.tests?.length) {
-  items.push(
-    {
-      header: "Info",
-      buttons: resolveRecording ? <StatusDropdown /> : null,
-      className: "replay-info",
-      component: <ReplayInfo />,
-      opened: !replayInfoCollapsed,
-      onToggle: () => setReplayInfoCollapsed(!replayInfoCollapsed),
-    },
-    {
+  items.push({
+    header: "Info",
+    buttons: resolveRecording ? <StatusDropdown /> : null,
+    className: "replay-info",
+    component: <ReplayInfo />,
+    opened: !replayInfoCollapsed,
+    onToggle: () => setReplayInfoCollapsed(!replayInfoCollapsed),
+  });
+
+  if (events.length > 0) {
```

Comment:
There is no `events` variable in scope here. The TypeScript error is because of this.